SwanCustomEnvironments/swancustomenvironments/serverextension.py

```diff
@@ -36,17 +38,31 @@ async def get(self):
             # The first get request will launch the execution of the makenv.sh script,
             # to create the environment
             makenv_process = self._launch_makenv()
-            SwanCustomEnvironmentsApiHandler.makenv_process = makenv_process
             log_file = open(self.LOG_FILE, "w")
+            SwanCustomEnvironmentsApiHandler.makenv_process = makenv_process
+            SwanCustomEnvironmentsApiHandler.writing_log = True
         else:
+            # Parent process has already been launched, so we will get its generated output
```

Comment:
It's not the parent process that has been launched, it is actually a child. You can say `The makenv.sh script process has already been launched, ...`